### ISSUE TITLE
MySQL error near largo_home_icon

### ISSUE THREAD

**Issue by benlk** [INN/Largo#1075]
```
[Thu Jan 14 12:44:36 2016] [error] [client 192.168.33.1] WordPress database error 
You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 
    '->posts} WHERE guid = 'http://vagrant.dev/wp-content/uploads/2015/12/rns-1170-11'
    at line 1 for query 
    SELECT ID FROM {$wpdb->posts} WHERE guid = 'http://vagrant.dev/wp-content/uploads/2015/12/rns-1170-1170x907-336x260.png' 
made by require('wp-blog-header.php'), 
require_once('wp-includes/template-loader.php'), 
include('/themes/largo/404.php'), 
get_header, 
locate_template, 
load_template, 
require_once('/themes/largo/header.php'), 
get_template_part, 
locate_template, 
load_template, 
require('/themes/largo/partials/nav-sticky.php'), 
largo_home_icon, 
referer: http://voiceofoc.vagrant.dev/
```

I'm not sure where that referrer is coming from, because this is on the RNS vagrant without subdomains.

https://github.com/INN/Largo/blob/master/inc/images.php#L32

**Comment by benlk** [comment]
This is showing up on most production sites, according to the wpengine logs.

**Comment by benlk** [comment]
Even in production.

**Comment by aschweigert** [comment]
is this our error or something broken in WP?

**Comment by benlk** [comment]
This is our error. Ryan said during review of #1114 that he thinks it's to do with something he changed in 0.5.4.

**Comment by benlk** [comment]
It's this line: https://github.com/INN/Largo/blob/master/inc/images.php#L39
We're using the logo's filename as a GUID, which is wrong.

**Comment by rclations** [comment]
fixed with 6ca1278
